src/components/ResultsItemDetails.vue

```diff
@@ -59,6 +59,11 @@ export default {
         ],
       parkName: 'Chatfield State Park'
     }
+  },
+  methods: {
+    favoritePark(parkName) {
```

Comment:
Once we're connected with the back-end, we'll need to change `parkName` to something like `parkDetails` since we'll want to pass the object containing all relevant data.

Reply:
👍